### DISCUSSION TITLE
发送请求后端成功发送但是前端接收到的是undefined的问题

### DISCUSSION THREAD

**Discussion by yifancy** [vbenjs/vue-vben-admin#2962] [General]
我用的是最新版本，然后就是接口是按照clone下来稍微做一下修改，后端可以发送数据，但是前端接收到的undefined，可是我用原生的axios是可以接收到数据的，求解答

**Comment by likui628** [comment]
需要看下后端返回的格式和vben是否一致，自己适当调整下
https://github.com/vbenjs/vue-vben-admin/blob/e31525a803ee5a7c9770b4eccebe91cc90f7aa60/src/utils/http/axios/index.ts#L53-L54

https://github.com/vbenjs/vue-vben-admin/blob/e31525a803ee5a7c9770b4eccebe91cc90f7aa60/types/axios.d.ts#L38-L43
